src/Components/Grid.js

```diff
@@ -0,0 +1,111 @@
+import React, {useState, useEffect} from 'react';
+import './Grid.scss';
+import Snake from './Snake';
+// Creating the instance of the snake
+let s = new Snake();
```

Comment:
Please, use a more descriptive variable name
`snake` should do it.

The other thing is that I would use a `const` instead of a `let`, because the variable won't be reassigned.